libs/vre/shared/app-resource-page/src/lib/resource.component.ts

```diff
@@ -39,10 +37,11 @@ export class ResourceComponent implements OnInit {
   @Input({ required: true }) resource!: DspResource;
   @Input({ required: true }) isDifferentResource!: boolean;
   representationsToDisplay!: ReadFileValue;
-  isCompoundNavigation: boolean | null = null;
+  isCompoundNavigation: boolean = false;
```

Comment:
isCompoundNavigation should be undefined (or null) by default

Reply:
It does not need to. The components display logic should be changed instead imho.